src/components/common/datagrid.js

```diff
@@ -115,29 +148,42 @@ function DataGridFactory(
       width: PropTypes.number.isRequired
     };
 
+    columnsSelector = props => props.columns;
+    hasGeojson = createSelector(this.columnsSelector, columns =>
+      columns.some(c => c.type === ALL_FIELD_TYPES.geojson)
+    );
+
     _cellRenderer = ({columnIndex, key, rowIndex, style}) => {
       const {columns, rows} = this.props;
 
+      const isLast = columnIndex === columns.length - 1;
+
       // rowIndex -1 because data rows start rendering at index 1 and we normalize back using the -1 param
       const className = `${rowIndex === 0
```

Comment:
you can use `classnames` to construct this, easier to follow
```js
      const className = classnames(`column-${columnIndex}`, {
        last: isLast,
        [`header-${columnIndex}`]: rowIndex === 0,
        [`row-${rowIndex-1}}`]: rowIndex > 0
      });```

Reply:
definitely better